File: Example/CoreData/SyncKitCoreDataExample/Model/QSEmployee+CoreDataProperties.m
//
//  QSEmployee+CoreDataProperties.m
//  SyncKitCoreDataExample
//
//  Created by Manuel Entrena on 08/05/2018.
//  Copyright © 2018 Manuel Entrena. All rights reserved.
//
//

#import "QSEmployee+CoreDataProperties.h"

@implementation QSEmployee (CoreDataProperties)

+ (NSFetchRequest<QSEmployee *> *)fetchRequest {
	return [NSFetchRequest fetchRequestWithEntityName:@"QSEmployee"];
}

@dynamic identifier;
@dynamic name;
@dynamic photo;
@dynamic sortIndex;
@dynamic company;

@end

File: Example/CoreData/SyncKitCoreDataExample/SyncKitCoreDataExample/Model/CoreData/QSEmployee+CoreDataProperties.swift
//
//  QSEmployee+CoreDataProperties.swift
//  
//
//  Created by Manuel Entrena on 11/06/2019.
//
//

import Foundation
import CoreData


extension QSEmployee {

    @nonobjc public class func fetchRequest() -> NSFetchRequest<QSEmployee> {
        return NSFetchRequest<QSEmployee>(entityName: "QSEmployee")
    }

    @NSManaged public var identifier: String?
    @NSManaged public var name: String?
    @NSManaged public var photo: Data?
    @NSManaged public var sortIndex: NSNumber?
    @NSManaged public var company: QSCompany?

}
